packages/angular/src/side-panel/side-panel.scss

```diff
@@ -0,0 +1,154 @@
+@import '../../../styles/src/globals/vars';
+@import '../../../styles/src/globals/typography';
```

Comment:
This should be imported from the package local vendor folder ... Which isn't published?? (https://unpkg.com/browse/@ai-apps/angular@2.146.0-next.59/) ... I'm checking that out ...

Reply:
If we can wait for #2617 this won't be an issue

Reply:
#2617 is in, if we update this it will be good to go 👍